enhancements/installer/crc.md

```diff
@@ -0,0 +1,93 @@
+---
+title: code-ready-containers
+authors:
+reviewers:
+approvers:
+creation-date: 2020-03-02
+last-updated: 2020-03-02
+status: implemented
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+superseded-by:
+---
+
+# Code Ready Containers (CRC)
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance, 
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this? 
+
+## Summary
+
+CRC functions by doing roughly....
+ 1.  Running a libvirt install, which is for ["development only"](https://github.com/openshift/installer/blob/ffc34e32fe4a71560f59312384daa87b401d6ec9/README.md) meaning OpenShift development only.
+ 2.  On a single node cluster, which is below the minimum of [three that are required](https://github.com/openshift/installer/blob/c904277e59dd947a8884265b2511034b05c38644/upi/openstack/inventory.yaml#L35) for a functional cluster.
```

Comment:
I would claim it is functional. The result can run pods, builds, store images, etc. So this seems like an incorrect statement.  The cluster does not meet our support requirements. But that doesn't mean it doesn't function.

Reply:
I think the is the operator logic that does not function properly as the operators were not designed to fully support one node clusters. The result is that CRC apply overrides for CVO and tweak the operator configs in unsupported way and as the operators are getting more clever we see a conflict.

Reply:
The argument is that a single node cluster is needed. Plus we likely need to make changes to the operator spec so that an operator can be used in a "dev mode" which would strip out things like HA which are largely unnecessary in dev. Understood that this isn't the situation today.

Reply:
HA is not some nice additional feature one  can just strip, it is a core assumption of all the deployment logic. For example we rely on HA to safely apply any configuration change. There is no fallback if anything goes wrong. The cluster will be dead immediately in single node mode.